edx_proctoring/templates/proctoring/proctoring_launch_callback.html

```diff
@@ -161,13 +161,22 @@
 
     <div class="steps-container">
       <div class="body-container">
-        <h3>{% blocktrans %} Your Proctoring Session Is Ready {% endblocktrans %}</h3>
-        <p>{% blocktrans %} Your proctoring session has started. From this point in time, you must adhere to the online proctoring rules to pass the proctoring review for your exam. Return to the courseware window now and start your exam. {% endblocktrans %}</p>
+        <h3>{% blocktrans %} Your Proctoring Session Has Started {% endblocktrans %}</h3>
+        <p>{% blocktrans %} From this point in time, you must follow the <a href="http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/sfd_proctoredexams/SFD_OnlineProctoringPolicy.html">online proctoring rules</a> to pass the proctoring review for your exam. {% endblocktrans %}</p>
 
         <div class="alert">
-          {% blocktrans %} Do not close this window. Leave it open in the background as you take your exam. {% endblocktrans %}
+          {% blocktrans %}
+            If you close this window before you finish your exam, your proctoring session ends,
+            and you might fail the proctoring review for your exam.
+          {% endblocktrans %}
         </div>
-        <h5> {% blocktrans %} Remember to abide by the guidelines and rules of your proctored exam, including: {% endblocktrans %}</h5>
+        <h5>
+          {% blocktrans %}
+            Return to the edX course window to start your exam. When you have finished your exam and
+            have marked it as complete, you can close this window to end the proctoring session
+            and upload your proctoring session data for review."
+          {% endblocktrans %}
+        </h5>
         <p>
           {% blocktrans %}
             Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
```

Comment:
Please delete the Latin placeholder text. Sorry if I missed this. By code freeze, we should make sure there are no instances of "Lorem ipsum..." left :-) Thanks!